meson.build

```diff
@@ -10,8 +10,9 @@ project('mogwai','c',
 
 # Dependency on GLib. We depend on 2.57.1, which isn’t packaged yet. When it
 # is packaged in Debian Unstable, we can drop this subproject dependency (FIXME).
-libglib_dep = dependency('glib-2.0', version: '>= 2.57.1',
-                         fallback: ['glib-2.0', 'libglib_dep'])
+# FIXME: Endless’ GLib 2.54.2 has the necessary backported patches, so we drop
```

Comment:
This is a wrap file, right?

I see that we can't just tell Meson not to use any wraps because we want gsystemservice, but if it would be helpful to do so I think we could do:

```
meson setup --wrap-mode nofallback --force-fallback-for gsystemservice
```

Not sure if that would work though.

In any case we don't need this patch because GLib on EOS master is 2.70.4-1bem1.

Reply:
It seemed simpler to keep the patch rather than risk breaking things. The patch can be dropped once upstream has subproject, which is something I didn’t want to think about right now